server/controllers/coordinates.js

```diff
@@ -0,0 +1,10 @@
+const axios = require('axios');
+
+const getCoordinates = async (addressQuery) => {
```

Comment:
Recommend deprecating API key and relocating new one to .env file in the future

Reply:
good call on parcel files. So I can delete that whole dist file correct?
and yea the API stuff is through Miguel, we'll have to get him to generate a new one